avae/loss.py

```diff
@@ -90,7 +90,9 @@ class AVAELoss:
 
     """
 
-    def __init__(self, device, beta, gamma=1, lookup_aff=None, recon_fn="MSE"):
+    def __init__(
+        self, device, beta, epochs, gamma=1, lookup_aff=None, recon_fn="MSE"
```

Comment:
Don't know if I am missing something here, but if `epochs` parameter is not actually used in the constructor, is it actually needed?